pkg/build/archive.go

```diff
@@ -103,7 +103,7 @@ func Archive(manifest model.Manifest) error {
 			istioctlBinary += ".exe"
 			istioctlDest += ".exe"
 		}
-		if err := util.CopyFile(path.Join(manifest.RepoOutDir("istio"), istioctlBinary), path.Join(out, "bin", istioctlDest)); err != nil {
+		if err := util.CopyFile(path.Join(os.Getenv("TARGET_OUT"), "release", istioctlBinary), path.Join(out, "bin", istioctlDest)); err != nil {
```

Comment:
this does not seem right, but maybe I am understanding wrong. This environment variable is only set in istio repo not here.

And as should always be copying Linux stuff here

Reply:
TARGET_OUT is set in the docker run command in the common Makefile. Maybe it's a misunderstanding by me on how the building container should be `docker run` (I'm basically doing what the common Makefile does).

Right now I'm chasing an issue in the validate where it seems there is a missing _proxyv2.tar.gz_, so could certainly be something wrong with changes as they are now.